### PR TITLE
Agent RPM installtion scripts

### PR DESCRIPTION
**Pull request by jakub-dzon**
This PR adds scripts and description for RPM-based Flotta agent installation.

### CODE REVIEW

File: hack/config.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+log-level = "info"
+cert-file = "/etc/pki/consumer/cert.pem"
+key-file = "/etc/pki/consumer/key.pem"
+ca-root = ["/etc/pki/consumer/ca.pem"]
+path-prefix = "api/flotta-management/v1"
+protocol = "http"
```
Comment (pkliczewski) [inline hack/config.toml:6]: I know it is not for this PR but should we change the protocol to https?

**Review by pkliczewski — COMMENTED** [review]
Looks good to me

File: hack/config.toml (hunk 1 above)
Comment (jakub-dzon) [reply hack/config.toml:6]: It is, in fact, https - `http` is just a confiig keyword for yggdrasil (`mqtt` vs. `http`).

**Review by masayag — APPROVED** [review]

**Review by eloycoto — APPROVED** [review]

File: hack/install-agent-dnf.sh.template [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<CERT_PEM>
+EOF
+
+cat <<EOF > /etc/yggdrasil/config.toml
+<CONFIG_TOML>
+EOF
+
+systemctl disable --now firewalld.service
+systemctl enable --now nftables.service
+systemctl enable --now podman.service
```
Comment (machacekondra) [inline hack/install-agent-dnf.sh.template:68]: no need for this line

**Pull request merged**